### PR TITLE
Add simple queries toolbar

### PR DESCRIPTION
**Pull request by nenadjaja**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -1,5 +1,14 @@
 import * as React from 'react'
 
+type SavedQuery = {
+  id: string
+  name: string
+  query: string
+  default?: boolean
+  subgraphId?: number
+  versionoId?: string
```
Comment (Jannis) [inline index.d.ts]: typo: `versionoId` -> `versionId`

File: src/components/SavedQueriesToolbar.js [hunk 2]
```diff
@@ -0,0 +1,556 @@
...
+    isOwner: PropTypes.bool,
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      open: props.isActionsMenuOpen,
+      selectedQueryName:
+        props.selectedQueryName &&
+        this.findSelectedQuery(props.queries, props.selectedQueryName)
```
Comment (Jannis) [inline src/components/SavedQueriesToolbar.js]: `findSelectedQuery` is probably cheap enough to run a few times. We could consider to call it only once above the state assignment and then use it down here.

File: src/components/SavedQueriesToolbar.js [hunk 3]
```diff
@@ -0,0 +1,556 @@
...
+          ? this.findSelectedQuery(props.queries, props.selectedQueryName).name
+          : this.defaultQuery(props.queries).name,
+      selectedQueryObj:
+        props.selectedQueryName &&
+        this.findSelectedQuery(props.queries, props.selectedQueryName)
```
Comment (Jannis) [inline src/components/SavedQueriesToolbar.js]: Same here

File: src/components/SavedQueriesToolbar.js [hunk 4]
```diff
@@ -0,0 +1,556 @@
...
+        default: false,
+        deleteDefault: false,
+      },
+    }
+
+    // Store selected query, so when cancel is clicked we can revert it
+    this.selectedQueryName = this.defaultQuery(props.queries).name
+    this.selectedQueryObj =
+      props.selectedQueryName &&
+      this.findSelectedQuery(props.queries, props.selectedQueryName)
```
Comment (Jannis) [inline src/components/SavedQueriesToolbar.js]: And here

**Review by Jannis — CHANGES_REQUESTED** [review]
Just a few small comments. It's hard to assess whether the code is doing the right things without comments, especially with all these success/error state flags. But like we said on the call earlier — we'll fix bugs as we spot them.

File: index.d.ts (hunk 1 above)
Comment (nenadjaja) [reply index.d.ts]: Good catch! I actually don't use it anywhere, so I'll remove it

**Comment by nenadjaja** [comment]
> Just a few small comments. It's hard to assess whether the code is doing the right things without comments, especially with all these success/error state flags. But like we said on the call earlier — we'll fix bugs as we spot them.

Yeah, I know. I'll add some comments. I am generally against adding too many comments, as they could clutter the code, and the code should be readable enough to speak for itself :)

**Review by Jannis — APPROVED** [review]

**Pull request merged**